InformationScripting/CMakeLists.txt

```diff
@@ -148,8 +148,18 @@ add_library(InformationScripting SHARED
 	src/query_framework/QueryRuntimeException.cpp
 	src/query_framework/HighlightOverlayStyle.cpp
 	src/queries/Yield.cpp
+	scripts/color.py
+	scripts/filterCasts.py
+	scripts/heatmapDataGen.py
+	scripts/not.py
+	scripts/sink.py
+	scripts/debug.py
+	scripts/filterMultiplications.py
+	scripts/ifToWhile.py
+	scripts/profileSource.py
+	scripts/toHtml.py
 )
 
 target_link_libraries(InformationScripting ${PYTHON_LIBRARIES} ${Boost_LIBRARIES})
 envision_plugin(InformationScripting OOInteraction OODebug)
-install(DIRECTORY scripts/ DESTINATION ${BUILD_DIR}/scripts)
\ No newline at end of file
+install(DIRECTORY scripts/ DESTINATION ${BUILD_DIR}/scripts)
```

Comment:
Because of the other change you did in the common cmake file, you no longer need this line